### ISSUE TITLE
compiler error on example code in readme of version 0.7.40

### ISSUE THREAD

**Issue by noguxun** [rkyv/rkyv#356]
If I compile below code (same as readme of 0.7.40), I got error like 

```
    |
565 |     #[archive_attr(derive(CheckBytes, Debug))]
    |                           ^^^^^^^^^^ use of undeclared crate or module `bytecheck`
    |
    = note: this error originates in the derive macro `CheckBytes` (in Nightly builds, run with -Z macro-backtrace for more info)
```

```
use rkyv::{Archive, Deserialize, Serialize};
// bytecheck can be used to validate your data if you want
use rkyv::CheckBytes;

#[derive(Archive, Deserialize, Serialize, Debug, PartialEq)]
// This will generate a PartialEq impl between our unarchived and archived types
#[archive(compare(PartialEq))]
// To use the safe API, you have to derive CheckBytes for the archived type
#[archive_attr(derive(CheckBytes, Debug))]
struct Test {
    int: u8,
    string: String,
    option: Option<Vec<i32>>,
}

let value = Test {
    int: 42,
    string: "hello world".to_string(),
    option: Some(vec![1, 2, 3, 4]),
};

// Serializing is as easy as a single function call
let bytes = rkyv::to_bytes::<_, 256>(&value).unwrap();

// Or you can customize your serialization for better performance
// and compatibility with #![no_std] environments
use rkyv::ser::{Serializer, serializers::AllocSerializer};

let mut serializer = AllocSerializer::<0>::default();
serializer.serialize_value(&value).unwrap();
let bytes = serializer.into_serializer().into_inner();

// You can use the safe API for fast zero-copy deserialization
let archived = rkyv::check_archived_root::<Test>(&bytes[..]).unwrap();
assert_eq!(archived, &value);

// Or you can use the unsafe API for maximum performance
let archived = unsafe { rkyv::archived_root::<Test>(&bytes[..]) };
assert_eq!(archived, &value);

// And you can always deserialize back to the original type
let deserialized: Test = archived.deserialize(&mut rkyv::Infallible).unwrap();
assert_eq!(deserialized, value);
```

**Comment by djkoloski** [comment]
Ah sorry about that, we'll be re-exporting bytecheck in the future to prevent this from happening. I'll correct the README.

**Comment by djkoloski** [comment]
Fixed by b351ede3e6460230c6f37ad9877b16892094116e. You'll need to add `bytecheck` to your direct dependencies at version 0.6.8 or later.
